.pre-commit-config.yaml

```diff
@@ -246,6 +246,12 @@ metastore_browser/templates/.*\\.html$|.*\\.jinja2"
         entry: "pydevd.*settrace\\("
         pass_filenames: true
         files: \.py$
+      - id: dont-use-safe-filter
+        language: pygrep
+        name: Don't use safe in templates
+        description: the Safe filter is error-prone, use Markup() in code instead
+        entry: "\\|\\s*safe"
+        pass_filenames: true
```

Comment:
Should we check only html files?

Reply:
Good idea.

Reply:
https://github.com/apache/airflow/pull/9180/commits/d22084c186b6d50fc60ec907a2bf7d91f4351d16